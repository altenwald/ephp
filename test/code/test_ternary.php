<?php

$a = ($a < 5) ? 0 : 5;
print "$a\n";

# PHP 7
var_dump($a ?? "no value");
var_dump($b ?? "no value");
