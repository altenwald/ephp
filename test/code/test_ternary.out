
Notice: Undefined variable: a in {{CWD}}/test/code/test_ternary.php on line 3
0
int(0)
string(8) "no value"
